guides/doc-Release_notes/topics/katello-4.4.0.adoc

```diff
@@ -2,87 +2,119 @@
+
+=== Container
+ * Docker download policy test failure (https://projects.theforeman.org/issues/34295[#34295], https://github.com/Katello/katello.git/commit/d4f2976f4d28e574d291dcbb793afa02b7b8a363[d4f2976f])
+
 === Hosts
  * Incorrect layout of new host details overview cards (https://projects.theforeman.org/issues/34258[#34258], https://github.com/Katello/katello.git/commit/bafada6dd069a3f1715b936cf9d1053a94b06bd9[bafada6d])
  * "Confirm services restart" modal window grammatically does not respect that multiple systems are selected for a reboot (https://projects.theforeman.org/issues/34037[#34037], https://github.com/Katello/katello.git/commit/30e6fe7d91d6296bb02830e4afaa3f66bee38795[30e6fe7d])
 
 === Tooling
  * faraday 1.9 and greater breaks tests (https://projects.theforeman.org/issues/34229[#34229], https://github.com/Katello/katello.git/commit/64b10e968117c79a441e3f9f1f253605722b2557[64b10e96])
+ * support new foreman tasks (https://projects.theforeman.org/issues/34097[#34097], https://github.com/Katello/katello.git/commit/d10534ae36aef9843a7a200cf651062d0628e630[d10534ae])
  * Re-enable disabled Rubocop cops that were turned off when fixing Rubocop Jenkins failure step (https://projects.theforeman.org/issues/31436[#31436], https://github.com/Katello/katello.git/commit/89230c545f5775c347d3bb259cacf89c665e5790[89230c54])
 
-=== Inter Server Sync
- *  [RFE] Need a way to sync from a specific content view lifecycle environment of the upstream organization (https://projects.theforeman.org/issues/34144[#34144], https://github.com/Katello/katello.git/commit/0cda22e6c989d5897bd19b936dead485d7290b73[0cda22e6])
- * RH Repos page should not access CDN on 'disconnected' mode (https://projects.theforeman.org/issues/33951[#33951], https://github.com/Katello/katello.git/commit/3e8f26961ff2a84a1de0b5dea197ea49a514d391[3e8f2696], https://github.com/Katello/katello.git/commit/b09ddee2f2c395c86f2f8c6cc862cc680f9debea[b09ddee2])
+=== Foreman Proxy Content
+ * stop using 'mirror=true' for smart proxy rpm repo syncs  (https://projects.theforeman.org/issues/34216[#34216], https://github.com/Katello/katello.git/commit/bc1bcf41a559006e3ac96b9cb1ef0b24a0be9e19[bc1bcf41])
+
+=== Hammer
+ * hammer content-view component list does not list content-view ID (https://projects.theforeman.org/issues/34174[#34174], https://github.com/Katello/hammer-cli-katello.git/commit/701d938830cc54357bafe19a4be6c857f58cc9dd[701d9388])
 
 === Lifecycle Environments
- * Katello should notify about published content view while removing Lifecycle environment (https://projects.theforeman.org/issues/33978[#33978], https://github.com/Katello/katello.git/commit/b8ac863c19f28d000ecd503e5cba359639240dd3[b8ac863c])
+ * Red Hat Satellite should notify about published content view while removing Lifecycle environment (https://projects.theforeman.org/issues/33978[#33978], https://github.com/Katello/katello.git/commit/b8ac863c19f28d000ecd503e5cba359639240dd3[b8ac863c])
```

Comment:
Looks like this was manually modified instead of the original issue. With Foreman we've always modified the original issue and then regenerated.